src/app/features/group/single-group/single-group.component.html

```diff
@@ -0,0 +1,21 @@
+<ng-container *ngIf="singleGroup$ | async as singleGroup">
+  <div class="main-container">
+    <header class="blue-container">
+      <h2 class="font-heading-2">{{ singleGroup.name }}</h2>
+    </header>
+    <div class="group-info">
+      <div class="group-info__users">
+        <h3 class="users-number font-heading-3">Członkowie grupy ({{ singleGroup.groupMembers.length }})</h3>
+        <ul class="users-list">
+          <li class="font-text-1 users-list__item" *ngFor="let member of singleGroup.groupMembers.slice(0, 3)">
```

Comment:
Może z `singleGroup.groupMembers.slice(0, 3)` zrobić gettera w komponencie? Chyba będzie trochę czytelniej

Reply:
`singleGroup` jest z async więc ciężko będzie zrobić get na to bo musiałbym się subskrybować w componencie